src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3348,6 +3364,11 @@ void MusicXMLParserDirection::wedge(const QString& type, const int number,
         if (niente == "yes") {
             h->setHairpinCircledTip(true);
         }
+        QColor color = QColor::Invalid;
+        color.setNamedColor(_e.attributes().value("color").toString());
```

Comment:
`const QColor color { _e.attributes().value("color").toString() };`

Here and all similar occurences